src/steamship/app/lambda_handler.py

```diff
@@ -50,33 +55,30 @@ def _handler(event: Dict, _: Dict = None) -> Response:
             logging.exception(ex)
             return Response.error(
                 code=HTTPStatus.INTERNAL_SERVER_ERROR,
-                message="Handler was unable to initialize plugin/app.",
+                prefix=error_prefix,
+                message=f"Unable to initialize plugin/app. {ex}",
```

Comment:
Do you want to put the exception in the message and return it in the exception? Given what you did below in response.py, it seems like that will duplicate the exception string.